src/traces/sankey/attributes.js

```diff
@@ -185,6 +190,21 @@ module.exports = overrideAll({
             role: 'info',
             description: 'A numeric value representing the flow volume value.'
         },
+        hoverinfo: {
+            valType: 'enumerated',
+            values: ['all', 'none', 'skip'],
+            dflt: 'all',
+            role: 'info',
+            description: [
+                'Determines which trace information appear when hovering links.',
+                'If `none` or `skip` are set, no information is displayed upon hovering.',
+                'But, if `none` is set, click and hover events are still fired.'
+            ].join(' ')
+        },
+        hoverlabel: fxAttrs.hoverlabel, // needs editType override,
         description: 'The links of the Sankey plot.'
     }
 }, 'calc', 'nested');
+// hide unsupported top-level properties from plot-schema
+attrs.hoverinfo = undefined;
```

Comment:
Oh so, trace-level `hoverinfo` is no longer available for sankey traces? Luckily it never worked so this isn't breaking change. 

Does this PR close https://github.com/plotly/plotly.js/issues/3097 ?

Reply:
It wouldn't close it. 

Issue #3097 is about supporting `hoverinfo` options other than the basic `skip` and `none`. Although the current doc say that we support many:
> Any combination of "label", "text", "value", "percent", "name" joined with a "+" OR "all" or "none" or "skip"

none actually worked except `all`. This PR is about slely dealing witk `skip` and `none` which are really important (ie. whether to hide the hoverlabels and whether to emit hover events at the same time).

Reply:
Good to know. Thanks!